engine/installation/linux/oracle.md

```diff
@@ -45,18 +53,18 @@ or volumes you created using the older version of Docker are preserved.
 
 You can install Docker in different ways, depending on your needs:
 
-- Most users
+- Users who do not require Docker support from Oracle
```

Comment:
I left it in the active voice, i.e. "these users do this" as opposed to "these users could do this". But you could add "can" back into this sentence as well.